docs/source/array-creation.rst

```diff
@@ -115,25 +115,52 @@ See :doc:`documentation on using dask.delayed with collections<delayed-collectio
 From Dask DataFrame
 -------------------
 
-You can create Dask arrays from Dask DataFrames using the ``.values`` attribute
-or the ``.to_records()`` method:
+At the most basic level, Dask DataFrames have a ``to_dask_array`` method:
```

Comment:
Might try to avoid saying what is "the most basic". For example, someone coming from pandas could think that the `.values` attribute is the most basic way to get an array from a DataFrame. Perhaps something like: 

```suggestion
There are several ways to create a Dask array from a Dask DataFrame. Dask DataFrames have a ``to_dask_array`` method:
```

Reply:
> Someone coming from pandas could think that the .values attribute 

The [pandas docstring for `.values`][1] says:

> **Warning: We recommend using `DataFrame.to_numpy()` instead.**

I used "at the most basic level" in the spirit of this warning. But, I'm not seeing much of a difference looking at the source (`np.asarray` or `df.values` is mapped to each chunk).

[1]:https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.values.html